mage_ai/server/server.py

```diff
@@ -420,7 +419,7 @@ def make_app():
         (r'/api/block_runs/(?P<block_run_id>\w+)/outputs', ApiBlockRunOutputHandler),
         (r'/api/block_runs/(?P<block_run_id>\w+)/logs', ApiBlockRunLogHandler),
         (r'/api/clusters/(?P<cluster_type>\w+)', ApiClustersHandler),
-        (r'/api/environments/(?P<env_type>\w+)', ApiEnvironmentsHandler),
+        (r'/api/clusters/(?P<cluster_type>\w+)/instances', ApiInstancesHandler),
```

Comment:
I wasn't sure if I should make this include the cluster name, but it would make the format of the endpoint kind of weird

Reply:
I think putting the cluster name in the payload is fine.